evennia/contrib/worldloader/example_tutorial_world/worlddata/csv/world_details.csv

```diff
@@ -0,0 +1,54 @@
+key,name,location,desc
```

Comment:
Excel on Windows and Numbers on Mac can handle these CSV files. When using Excel, the outputted CSV files must convert to UTF8.